base_continent/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Base Continent
+==============
+
+This module adds a *Continent* object (res.continent) and a link between continents and countries. It also adds a continent field on partners (automatically set if a country has been selected on the partner).
+
+Configuration
+=============
+
+Go to the menu *Sales > Configuration > Address Book > Localization > Continents* to see the list of continents and assign countries to continents.
+
+Usage
+=====
+
+You can now group partners per continent.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/8.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/partner-contact/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```

Comment:
@alexis-via Thanks for the contribution!
In the newest README template you do not need this line